script/Ciel/Day 5/05_01_CIEL05_3B.txt

```diff
@@ -469,57 +498,65 @@ I call out to her instinctively.
 [sha:853e1f276906f47b1d2651854af8205224f1d872]{
 -- Page 476, Offset 12119.
 -- 「…………その。先輩は、吸血鬼を殺す為に、この街にやって来たんですか？」
-"...Umm. Senpai, did you come to this city to kill vampires?"
+"Umm... Senpai, did you come to this city to kill vampires?"
 }
 [sha:8980180a2129fc5311ca72ecfcb2776e8a20fe08]{
 -- Page 477, Offset 12120.
 -- 「そういう事になりますね。もともと私はこの街の人間ではありません。貴方の学校に生徒として編入していたのも吸血鬼を殲滅する為です。すべてが終わればこの街を後にします」
-"That's how it is. I'm not originally from this city. I enrolled at your school for the purpose of exterminating that vampire. When everything is over, I'll leave this city."
+"That's how it is.
+ I'm not originally from this city.
+ Even enrolling at your school was all in service of exterminating vampires.
+ When everything is over, I'll leave this city and move on."
 }
 [sha:67ede1f32c65f3863fbfa25f7604875bd7031bdf]{
 -- Page 478, Offset 12121.
 -- 「その時、貴方の記憶を消去する事にしましょう。
-"Once that happens, I've decided that I will purge your memories.
+"When that times comes, I'll also erase your memories.
 }
 [sha:54ac8e98f31aeb0d60a1ec375d78906741c8a2fb]{
 -- Page 478, Offset 12122. Mods: @k, @e.
 -- それまでは見逃してあげますから、くれぐれも危険な行動は慎むように。
-Until then, I'll let you off the hook. Please refrain earnestly from dangerous activities.
+Until then, I'll let you off the hook.
+ Please make an honest effort to refrain from any dangerous activities.
 }
 [sha:e343060ef2bbb2ee1144994ccac8fa6e1a59fdbb]{
 -- Page 478, Offset 12123. Glued. Mods: @x.
 -- 次に私の視界に入れば障害物として処理しますから、そのつもりで」
-The next time I see you, I fully intend on dealing with you as a hindrance."
+ Be advised that the next time I see you,
+ I will consider you an obstacle, and deal with you accordingly."
 }
 [sha:fba54e2a8d4d36d4e400e20eb24ff9e530e5d169]{
 -- Page 479, Offset 12124.
 -- 冷徹な視線で、冷めきった声で、先輩は俺に警告した。
-With a level-headed gaze and a cold voice, Senpai warns me.
+With a level-headed gaze and a cold voice, Senpai issues me a warning.
 }
 [sha:1a9c71025b15c6f2151467c7568552577b0b460e]{
 -- Page 479, Offset 12125.
 -- 家に帰れ。
-Go home.
+She's telling me to go home.
```

Comment:
i think revert this line and the next but just put them in single quotes but could go either way tbh